src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdmCloseServiceTest.java

```diff
@@ -98,21 +98,7 @@ void generateCloseApplicationMarkdown(boolean appHasDoc, boolean appHasDetails,
                 .build())
         );
 
-        String expected = "Application Details\r\n"
-            + "|Decision | |\r\n"
-            + "|--|--|\r\n"
-            + "|Notification | Response Details|\r\n"
-            + "|Decision | decision|\r\n"
-            + "|Decision details | decision details|\r\n"
-            + "|Date | 23 December 2022|\r\n"
-            + "|Sent by | Tribunal|\r\n"
-            + "|Type of decision | type of decision|\r\n"
-            + (hasAdditionalInfo ? "|Additional information | additional info|\r\n" : "")
-            + (hasDoc ? "|Document|<a href=\"/documents/%s\" target=\"_blank\">document (TXT, 1MB)</a>|\r\n" : "")
-            + "|Decision made by | decision made by|\r\n"
-            + "|Name | made by full name|\r\n"
-            + "|Sent to | party notify|\r\n"
-            + "\r\n";
+        String expected = "Application Details\r\n";
```

Comment:
Is this test necessary now? It's being run 3 times with different values, but they don't seem to be needed

Reply:
I guess not, should I remove it?

Reply:
Sorry that was worded badly, a single test is valid but it doesn't need to be run 3 times now by the looks of it